ui/src/components/Log/index.jsx

```diff
@@ -276,12 +245,14 @@ export default class Log extends Component {
                 className:
                   FollowLogButtonProps && FollowLogButtonProps.className,
               }}
-              tooltipProps={{ title: follow ? 'Unfollow Log' : 'Follow Log' }}
+              tooltipProps={{
+                title: follow && stream ? 'Unfollow Log' : 'Follow Log',
+              }}
               variant="round"
               className={classNames({
-                [classes.followButtonFollowing]: follow,
+                [classes.followButtonFollowing]: follow && stream,
               })}
-              color={follow ? 'inherit' : 'secondary'}
+              color={follow && stream ? 'inherit' : 'secondary'}
               onClick={this.handleFollowClick}
               {...FollowLogButtonRest}>
               <ArrowDownBoldCircleOutlineIcon />
```

Comment:
For a log that is not streaming, we should be okay removing the `onScroll` props. I think you can do:

```jsx
onScroll={stream ? this.handleScroll : undefined}
```

Reply:
Alright.